Dockerfile-alpine.template

```diff
@@ -7,12 +7,12 @@ COPY requirements.txt /tmp/
 
 RUN set -ex \
     && apk add --no-cache --virtual .build-deps \
-        --repository http://dl-cdn.alpinelinux.org/alpine/v3.9/main \
+        --repository http://dl-cdn.alpinelinux.org/alpine/v3.11/main \
```

Comment:
Due to the moving pointer nature of Docker image tags, we are on Alpine 3.11.

Reply:
I'm in favor of pinning to `alpine3.11` in the `FROM` line as well, similarly to how we pin to `buster` in the `slim` variant. If we do that we might be able to remove `--repository http://dl-cdn.alpinelinux.org/alpine/v3.11/main` options altogether. I'm seeing the same output from `apk add` when running it

Reply:
I agree. FYI, the Alpine version pinned package repositories were added by https://github.com/azavea/docker-django/commit/6a5baa8d1c2a609dccd61c43952811624b83fc37#diff-5c587da9015d91ff9da4528825b5add6.

Should be fixed by:

- 81f3a885381abe1915d6f5ca904a913d387ff0a0
- cbc4759d5fa6b33f8394549795bd59ba002552b9
- dbd07639b97b05509956bcacdec0477a3966dfed